content/sensu-go/5.20/api/webconfig.md

```diff
@@ -0,0 +1,286 @@
+---
+title: "Web UI configuration API"
+linkTitle: "Web UI Configuration API"
+description: "The Sensu web API provides HTTP access to the active web UI configuration. This reference includes examples for returning the active web configuration and adding or updating the web configuration. Read on for the full reference."
+version: "5.20"
+product: "Sensu Go"
+menu:
+  sensu-go-5.20:
+    parent: api
+---
+
+- [The `/globals` API endpoint](#the-globals-api-endpoint)
```

Comment:
I apologize, I've last minute changed the endpoint. It's now `http://127.0.0.1:8080/api/enterprise/web/v1/globals`.

See: https://github.com/sensu/sensu-enterprise-go/pull/1026

Reply:
@jamesdphillips sorry, so did v1 change to v2 and globals changed to config?

Reply:
^ I changed it to `...web/v2/config` so just let me know if that's incorrect. Thank you for the heads up!

Reply:
Whoops-- I missed this comment. It is `/api/enterprise/web/v1/config`.

Sorry for the confusion.